controllers/flowcollector_controller.go

```diff
@@ -309,6 +309,12 @@ func (r *FlowCollectorReconciler) reconcileOperator(ctx context.Context, cmn *re
 			return err
 		}
 	}
+
+	desiredFlowDashboardCM := buildFlowMetricsDashboard()
+	if err := cmn.ReconcileConfigMap(ctx, desiredFlowDashboardCM); err != nil {
+		return err
+	}
```

Comment:
I think it makes sense to move that under the same condition as the health dashboard, in `if r.availableAPIs.HasSvcMonitor()` below? Is there a reason to not do that?